agent/serviceinfobroker/service_info_broker.go

```diff
@@ -84,12 +84,12 @@ func (sib *ServiceInfoBroker) GetInfoFromService(ctx context.Context, msg *agent
 	}
 }
 
-func (sib *ServiceInfoBroker) getMySQLInfo(ctx context.Context, dsn string, files *agentpb.TextFiles, id uint32) *agentpb.ServiceInfoResponse {
+func (sib *ServiceInfoBroker) getMySQLInfo(ctx context.Context, dsn string, files *agentpb.TextFiles, tlsSkipVerify bool, id uint32) *agentpb.ServiceInfoResponse {
```

Comment:
nit: Doesn't the linter yell at the line being too long? :)

Reply:
I dont see it between checks. Now it is 164 chars, I think lll is >180.

Reply:
You are right, it's < 170 :)

https://github.com/percona/pmm/blob/main/.golangci.yml#L57